docs/content_management/locations.md

```diff
@@ -90,8 +90,14 @@ Location visibility allows you to control which parts of the content tree are av
 ![Location visibility](content_management_visibility.png "Location visibility")
 
 Once a Content item is published, it cannot be un-published.
-Limiting visibility is the only way to withdraw content from the website without moving it to Trash.
-When the Location of a Content item is hidden, any access to it will be denied, preventing the system from displaying it.
+When the Location of a Content item is hidden, the system will not display it on the website front by default.
```

Comment:
"withdraw content from the website" is not precise enough.
"access to it will be denied" actually contradicts the fact that visibility in not permission-based